### PR TITLE
Fixes #9693 - make identifier information clear

### PR DESCRIPTION
**Pull request by ares**
let me know if you think I should mention NetBSD (agr) and OpenBSD (trunk) as well, I think it's long enough already

### CODE REVIEW

**Comment by unorthodoxgeek** [comment]
why is it in plain English? shouldn't this be translated? if not, it's good to merge

**Comment by domcleal** [comment]
It's inside a `_()` call so will be extracted for translation.

**Comment by dLobatog** [comment]
Merged as 38f4f118fd6e1a134f89e87326e210f803be0e92, thanks @ares
